src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -219,7 +219,8 @@ std::string KeyboardShortcuts::GetShortcutString(int32_t shortcut) const
 ScreenCoordsXY KeyboardShortcuts::GetKeyboardMapScroll(const uint8_t* keysState) const
 {
     ScreenCoordsXY screenCoords;
-    for (int32_t shortcutId = SHORTCUT_SCROLL_MAP_UP; shortcutId <= SHORTCUT_SCROLL_MAP_RIGHT; shortcutId++)
+    for (int32_t shortcutId = static_cast<int32_t>(Input::Shortcut::ScrollMapUp);
+         shortcutId <= static_cast<int32_t>(Input::Shortcut::ScrollMapRight); shortcutId++)
     {
         uint16_t shortcutKey = _keys[shortcutId];
         uint8_t scancode = shortcutKey & 0xFF;
```

Comment:
L228 this is not Input::Shortcut::Undefined but a different variable with the same value.